cli/internal/runcache/runcache.go

```diff
@@ -109,12 +109,12 @@ type TaskCache struct {
 
 // RestoreOutputs attempts to restore output for the corresponding task from the cache.
 // Returns true if successful.
```

Comment:
A comment here explaining the new return type would be nice. Error is self-explanatory.